tests/test_pydantic.py

```diff
@@ -4,6 +4,13 @@
 
 pydantic = pytest.importorskip("pydantic")
 
+try:
+    pydantic.version.version_info()
+    func_json = "model_dump_json"
+except:
+    pydantic.utils.version_info()
```

Comment:
what's the point of calling `version_info()` again?

Reply:
removed